doc/api/n-api.md

```diff
@@ -493,8 +493,9 @@ implementation can use to persist VM-specific state. This structure is passed
 to native functions when they're invoked, and it must be passed back when
 making N-API calls. Specifically, the same `napi_env` that was passed in when
 the initial native function was called must be passed to any subsequent
-nested N-API calls. Caching the `napi_env` for the purpose of general reuse is
-not allowed.
+nested N-API calls. `napi_env` can be used through the context with the module,
+that is to say, `napi_env` can not be used mixed up in  [`Worker`][] threads,
+and caching the `napi_env` for the purpose of general reuse is not allowed.
```

Comment:
Maybe we should just change the last sentence to this:

"Caching the `napi_env` for the purpose of general reuse, and passing the `napi_env` between instances of the same module running on different [`Worker`][] threads is not allowed."